game/src/core/components/DrawComponent.java

```diff
@@ -151,6 +154,28 @@ public void currentAnimation(final IPath... animationName) {
         }
     }
 
+    /**
```

Comment:
das muss mit in die klassenjavadocl


Bruacht es. `#currentAnimation` (also setter dann eigentlich noch)

Reply:
Hab die Klassenjavadoc dahingehend geupdated

`#currentAnimation` würde ich beibehalten, da die queue-logik jetzt im Drawsystem sitzt
außerdem nutzen einige Tests noch die Methode, ist aber fraglich, ob das noch so sinnig ist